### PR TITLE
Create tmp folder if not present

### PR DESCRIPTION
**Pull request by chashmeetsingh**
### Summary

All Apps may not have a tmp folder

### CODE REVIEW

**Comment by rafaelfranca** [comment]
Thank you for the pull request. We should do the same as the restart task.

**Comment by chashmeetsingh** [comment]
@rafaelfranca good?

**Comment by chashmeetsingh** [comment]
@rafaelfranca good?

File: railties/lib/rails/tasks/dev.rake [hunk 1]
```diff
@@ -1,14 +1,18 @@
 namespace :dev do
   desc 'Toggle development mode caching on/off'
   task :cache do
+    if not Dir.exist? 'tmp'
```
Comment (rafaelfranca) [inline railties/lib/rails/tasks/dev.rake]: We don't need the if here. `mkdir_p` already do it.

**Comment by chashmeetsingh** [comment]
@rafaelfranca now?

File: railties/lib/rails/tasks/dev.rake [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+  
     if File.exist? 'tmp/caching-dev.txt'
       File.delete 'tmp/caching-dev.txt'
       puts 'Development mode is no longer being cached.'
     else
       FileUtils.touch 'tmp/caching-dev.txt'
       puts 'Development mode is now being cached.'
-    end
-
+    end
```
Comment (rafaelfranca) [inline railties/lib/rails/tasks/dev.rake]: Could you please remove the extra whitespace that there is in the end of this line and the line below?

**Comment by chashmeetsingh** [comment]
done

File: railties/lib/rails/tasks/dev.rake [hunk 3]
```diff
@@ -8,7 +10,6 @@ namespace :dev do
       FileUtils.touch 'tmp/caching-dev.txt'
       puts 'Development mode is now being cached.'
     end
-
```
Comment (rafaelfranca) [inline railties/lib/rails/tasks/dev.rake]: Keep the blank line :smile:

**Comment by rafaelfranca** [comment]
Could you squash your commits?

**Comment by chashmeetsingh** [comment]
@rafaelfranca not able to squash

**Comment by rafaelfranca** [comment]
Why not?

**Comment by chashmeetsingh** [comment]
I did a rebase , commit then push still nothing updated here :(

**Comment by rafaelfranca** [comment]
you need to force push

**Comment by chashmeetsingh** [comment]
not happening :(

**Comment by chashmeetsingh** [comment]
@rafaelfranca good to go?

**Comment by rafaelfranca** [comment]
It still had some extra whitespaces but I merged and removed myself. Thanks!

**Comment by chashmeetsingh** [comment]
thanks mate!

**Pull request merged**